src/nano_contracts/deserializer.ts

```diff
@@ -22,6 +22,12 @@ class Deserializer {
    * @inner
    */
   deserializeFromType(value: Buffer, type: string): any {
+    if (type.endsWith('?')) {
```

Comment:
Do we have a list of types or modifiers?
If so I think we should add a link to it with the [see tag](https://jsdoc.app/tags-see).

Reply:
Another alternative to a `@see` tag, since we're using Typescript on this repository, would be to add a type for the `type` parameter.

It's a developer-friendly way to have this information always at hand, and static code validation, on compatible IDEs.

Reply:
As we discussed in private, we currently don't have this documentation link. Also, the list of possible types is not static, so the documentation link is better.

I created an issue so we can track this when we have the documentation link. https://github.com/HathorNetwork/hathor-wallet-lib/issues/649